PROShine/Views/MainWindow.xaml.cs

```diff
@@ -73,9 +73,20 @@ public MainWindow()
             Bot.TextboxRemoved += Bot_TextboxRemoved;
             
             InitializeComponent();
-            AutoReconnectSwitch.IsChecked = Bot.AutoReconnector.IsEnabled;
-            AvoidStaffSwitch.IsChecked = Bot.StaffAvoider.IsEnabled;
-            AutoEvolveSwitch.IsChecked = Bot.PokemonEvolver.IsEnabled;
+
+            AutoEvolveSwitch.IsChecked = UserSettings.AutoEvolve;
+            AvoidStaffSwitch.IsChecked = UserSettings.AvoidStaff;
+            AutoReconnectSwitch.IsChecked = UserSettings.AutoReconnect;
+
+            Bot.PokemonEvolver.IsEnabled = UserSettings.AutoEvolve;
+            Bot.StaffAvoider.IsEnabled = UserSettings.AvoidStaff;
+            Bot.AutoReconnector.IsEnabled = UserSettings.AutoReconnect;
+
+            if (!string.IsNullOrEmpty(UserSettings.LastScript))
+            {
+                MenuReloadScript.Header = "Reload " + Path.GetFileName(UserSettings.LastScript) + "\tCtrl+R";

```

Comment:
You can probably use `InputGestureText="Ctrl+R"` in the XAML instead of adding the string manually.

Reply:
Cool, I didn't know about that.